### ISSUE TITLE
Warning on selecting date from the infinte calender 

### ISSUE THREAD

**Issue by Parvinder1991** [clauderic/react-infinite-calendar#28]
Warning: Failed propType: Invalid prop `shouldHeaderAnimate` of type `string` supplied to `Header`, expected `boolean`. Check the render method of `InfiniteCalendar`.

Object { selectedDate: Object, shouldHeaderAnimate: ".0.0.1.0.0.0.0.0.2.0.0.0.0.2.0.0.1.1.1.0.$3-0.$Month-3.0.$Row-1.$day-6", layout: "portrait", theme: Object, locale: Object, scrollToDate: InfiniteCalendar/_this.scrollToDate(), setDisplay: InfiniteCalendar/_this.setDisplay(), display: "days" }

**Comment by clauderic** [comment]
Did you pass in a custom `shouldHeaderAnimate` prop of type `string`? If not, can you share some more details and some code?

**Comment by Parvinder1991** [comment]
No i didnt pass a custom shouldHeaderAnimate of type string ...here are some more details 

//for toggling calender 
toggleCalender() {
    switch(this.state.showcalendar){
      case true:
        this.setState({showcalendar: !this.state.showcalendar, today: this.state.today, minDate: this.state.minDate, maxDate: this.state.maxDate});
      break;
      case false:
        this.setState({showcalendar: !this.state.showcalendar, today: this.state.today, minDate: this.state.minDate, maxDate: this.state.maxDate});
      break;
    } 
  }
//calender Impelementation
const InfiniteCalendarComponent = React.createClass({
  getInitialState() {
    this.state = {};
    return this.state;
  },
  componentWillReceiveProps: function(nextProps) {
  this.setState({
    today: this.props.today, minDate: this.props.minDate, maxDate: this.props.maxDate
  });
},

  render() {
    return (
      <div>
        <InfiniteCalendar
          width={'100%'}
          height={300}
          selectedDate={this.state.today}
          min={this.state.today}
          minDate={this.state.minDate}
          keyboardSupport={true}
          max={this.state.maxDate}
          onSelect={this.props.setDate}
        />
      </div>
    );
  }
});

export default InfiniteCalendarComponent;

**Comment by clauderic** [comment]
Would you mind replicating this bug in a jsfiddle? Having a hard time wrapping my head around what exactly it is you're doing. You can fork this [fiddle](https://jsfiddle.net/clauderic/4rhn03do/)

**Comment by efegurkan** [comment]
It might be webpack specific BTW. I am also having the same issue when I tried to do. 
Nothing but this on a dummy page with a render on it. Also I installed it using node 5.12.0 npm 3.8.6.

Out of the topic: Had a hard time to set this up using webpack because it didn't wanted to include moment-range to the build. Then upgraded to node version above, It is solved kind of.

```javascript
import InfiniteCalendar from "react-infinite-calendar";

        <InfiniteCalendar
          min={new Date(1900, 0, 1)}
          minDate={new Date(1900, 0, 1)}
        />

```
